kod/object/passive/body.kod

```diff
@@ -196,11 +196,6 @@ messages:
 
    GetObjectFlags()
    {
-      if pbMob
```

Comment:
Was this removed intentionally?  It seems important and its removal could have a lot of side effects.

Reply:
It was, as with it the monster corpse is not targetable.

And yes, there are side effects I didn't catch. Notably, animate can now be cast upon these monster corpses and that's outside of the scope of this change. That I can address, but removing this flag also means that corpses (possibly many) will show up in the Look dialog list. That seems undesirable to me.

Going to drop this into draft mode while I consider next steps. Open to suggestions.